src/sparseml/pytorch/object_detection/detect.py

```diff
@@ -0,0 +1,365 @@
+# YOLOv5 🚀 by Ultralytics, GPL-3.0 license
```

Comment:
not sure if we need necessarily the pytorch detect script in sparseml src since there will be an annotate script in deepsparse focused on the onnx exports

Reply:
Can delete now or keep up until that pathway exists. Could it have some utility for user who are familiar with the yolov5 interface as an easy way to gauge/qualify performance? Potentially useful for the purpose of comparison between native and deepsparse performance (for the user). 

These are minor uses, so may be better off removing for simplification anyway

Reply:
would prefer to delete this file for now to 1) limit scope of what is included here and 2) focus on building the deepsparse.yohlov5.annotate pathway